src/coreclr/tests/runtest.py

```diff
@@ -1635,5 +1638,8 @@ def main(args):
 ################################################################################
 
 if __name__ == "__main__":
+    print(" !!! runtest.py: main")
```

Comment:
This looks like temporary instrumentation, can you please remove it before merging this in?